lib/core/datatable/data/object-datatable-adapter.ts

```diff
@@ -77,6 +77,10 @@ export class ObjectDataTableAdapter implements DataTableAdapter {
         this.rowsChanged = new Subject<Array<DataRow>>();
     }
 
+    getType(_row: DataRow, col: DataColumn): string {
```

Comment:
as above, does not really make sense introducing a method with parameters it does not even use